doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -929,6 +802,10 @@ Quitting and debugging
       If the given :n:`@command` succeeds, then :n:`Fail @command`
       fails with this error message.
 
+.. note:: :cmd:`Time`, :cmd:`Redirect`, :cmd:`Timeout` and :cmd:`Fail` are
+   :production:`control_command`\s; for these commands, goal selectors and
+   attributes must be placed after the command name and before the inner
+   :n:`@sentence``.  For example, `Time :1 auto.` or `Time Timeout 10 :1 auto.`.
```

Comment:
```suggestion
.. note::

   :cmd:`Time`, :cmd:`Redirect`, :cmd:`Timeout` and :cmd:`Fail` are
   :production:`control_command`\s. When specified, attributes and goal
   selectors are part of the :n:`@sentence` argument, and thus come after
   the control command prefix and before the inner command or tactic. For
   example: `Time #[ local ] Definition foo := 0.` or `Fail Timeout 10 all: auto.`
```

I removed the last period which was confusing (not clear it's not part of the second example).

Reply:
How is this?  (Adding "For these commands" and moving "when specified" compared to your suggestion)

```
   :cmd:`Time`, :cmd:`Redirect`, :cmd:`Timeout` and :cmd:`Fail` are
   :production:`control_command`\s. For these commands, attributes and goal
   selectors, when specified, are part of the :n:`@sentence` argument, and thus come after
   the control command prefix and before the inner command or tactic. For
   example: `Time #[ local ] Definition foo := 0.` or `Fail Timeout 10 all: auto.`
```

Reply:
LGTM!